be/test/util/path_util_test.cpp

```diff
@@ -0,0 +1,85 @@
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+#include "util/path_util.h"
+
+#include <string>
+#include <vector>
+
+#include <gtest/gtest.h>
+
+#include "common/config.h"
+#include "util/logging.h"
+
+
+using std::string;
+using std::vector;
+
+namespace doris {
+
+TEST(TestPathUtil, BaseNameTest) {
+    ASSERT_EQ(".", path_util::base_name(""));
+    ASSERT_EQ(".", path_util::base_name("."));
+    ASSERT_EQ("..", path_util::base_name(".."));
+    ASSERT_EQ("/", path_util::base_name("/"));
+    ASSERT_EQ("/", path_util::base_name("//"));
+    ASSERT_EQ("a", path_util::base_name("a"));
+    ASSERT_EQ("ab", path_util::base_name("ab"));
+    ASSERT_EQ("ab", path_util::base_name("ab/"));
+    ASSERT_EQ("cd", path_util::base_name("ab/cd"));
+    ASSERT_EQ("ab", path_util::base_name("/ab"));
+    ASSERT_EQ("ab", path_util::base_name("/ab///"));
+    ASSERT_EQ("cd", path_util::base_name("/ab/cd"));
+}
+
+TEST(TestPathUtil, DirNameTest) {
+    ASSERT_EQ(".", path_util::dir_name(""));
+    ASSERT_EQ(".", path_util::dir_name("."));
+    ASSERT_EQ(".", path_util::dir_name(".."));
+    ASSERT_EQ("/", path_util::dir_name("/"));
+    ASSERT_EQ("//", path_util::dir_name("//"));
+    ASSERT_EQ(".", path_util::dir_name("a"));
+    ASSERT_EQ(".", path_util::dir_name("ab"));
+    ASSERT_EQ(".", path_util::dir_name("ab/"));
+    ASSERT_EQ("ab", path_util::dir_name("ab/cd"));
+    ASSERT_EQ("/", path_util::dir_name("/ab"));
+    ASSERT_EQ("/", path_util::dir_name("/ab///"));
+    ASSERT_EQ("/ab", path_util::dir_name("/ab/cd"));
+}
+
+TEST(TestPathUtil, SplitPathTest) {
+    using Vec = vector<string>;
+    ASSERT_EQ(Vec({"/"}), path_util::split_path("/"));
+    ASSERT_EQ(Vec({"/", "a", "b"}), path_util::split_path("/a/b"));
+    ASSERT_EQ(Vec({"/", "a", "b"}), path_util::split_path("/a/b/"));
+    ASSERT_EQ(Vec({"a", "b"}), path_util::split_path("a/b"));
+    ASSERT_EQ(Vec({"."}), path_util::split_path("."));
+    ASSERT_EQ(Vec(), path_util::split_path(""));
+}
+
+} // namespace doris
+
+int main(int argc, char* argv[]) {
+    std::string conffile = std::string(getenv("DORIS_HOME")) + "/conf/be.conf";
+    if (!doris::config::init(conffile.c_str(), false)) {
+        fprintf(stderr, "error read config file. \n");
+        return -1;
+    }
+    doris::init_glog("be-test");
```

Comment:
can you remove these codes to make it run without conf?

Reply:
Done.